csrankings.csv

```diff
@@ -19971,6 +19973,7 @@ Xiaofeng Wang 0006,Indiana University,http://www.informatics.indiana.edu/xw7,u4d
 Xiaogang Jin 0001,Zhejiang University,http://www.cad.zju.edu.cn/home/jin,yryOvLwAAAAJ
 Xiaogang Jin 0002,Zhejiang University,http://mypage.zju.edu.cn/0096364,tYqG3CAAAAAJ
 Xiaogang Ma,University of Idaho,http://www.uidaho.edu/engr/departments/cs/our-people/faculty/xiaogang-ma,o4v-pvoAAAAJ
+Xiaogang Wang,Chinese University of Hong Kong,http://www.ee.cuhk.edu.hk/~xgwang/,B5JgjsAAAAJ

```

Comment:
None of these folks are listed here, making their eligibility for inclusion unclear: https://www.cse.cuhk.edu.hk/en/people/faculty/by-profile